app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,270 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.core.widget.NestedScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:forceDarkAllowed="false"
+    android:orientation="horizontal"
+    android:background="#191926"
+    tools:context=".MovieDetailsActivity">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ImageView
+            android:id="@+id/Mask"
+            android:contentDescription="@string/movie_title"
+            android:layout_width="match_parent"
```

Comment:
[Important: MATCH_PARENT is not recommended for widgets contained in a ConstraintLayout](https://developer.android.com/reference/androidx/constraintlayout/widget/ConstraintLayout)